src/test/java/com/mparticle/kits/GoogleAnalyticsFirebaseKitTest.java

```diff
@@ -44,6 +45,8 @@
     GoogleAnalyticsFirebaseKit kitInstance;
     FirebaseAnalytics firebaseSdk;
     Random random = new Random();
+    private static Logger LOGGER = Logger.getLogger("InfoLogging");
```

Comment:
what's this for? looks like it's unused

Reply:
I think I added this when I was testing to log things out, but forgot to remove.  I'll take that out now.